Bottom: 04bed845babf41ff9c2b756231a302af6bd4ce1d
Top:    6f59c907c8ddc703a12a9928cd807d47745cb9e3
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-23 18:13:19 -0400

Add a reset function


---

diff --git a/arch/arm/mach-lpc31xx/board-ea313x.c b/arch/arm/mach-lpc31xx/board-ea313x.c
index b7864a4..0e537ec 100644
--- a/arch/arm/mach-lpc31xx/board-ea313x.c
+++ b/arch/arm/mach-lpc31xx/board-ea313x.c
@@ -47,6 +47,7 @@
 #include <mach/gpio.h>
 #include <mach/i2c.h>
 #include <mach/board.h>
+#include <mach/system.h>
 
 static struct lpc313x_mci_irq_data irq_data = {
 	.irq = IRQ_SDMMC_CD,
@@ -735,6 +736,7 @@ MACHINE_START(EA313X, "NXP EA313X")
 	.init_irq	= lpc313x_init_irq,
 	.timer		= &lpc313x_timer,
 	.init_machine	= ea313x_init,
+	.restart	= arch_reset,
 MACHINE_END
 #endif
